new_pages/basics.qmd

```diff
@@ -364,7 +360,7 @@ age_pyramid(linelist, "age_cat5", "gender")
 **A more complex `age_pyramid()` command might include the *optional* arguments to:**
 
 -   Show proportions instead of counts (set `proportional = TRUE` when the default is `FALSE`)\
--   Specify the two colors to use (`pal =` is short for "palette" and is supplied with a vector of two color names. See the [objects](#objectstructure) page for how the function `c()` makes a vector)
+-   Specify the two colors to use (`pal =` is short for "palette" and is supplied with a vector of two color names. See the [objects](https://epirhandbook.com/new_pages/basics.html#objects) page for how the function `c()` makes a vector)
```

Comment:
@arranhamlet is there a reason this link was hard-coded to URL instead of to the page's tag?

Reply:
Missed this in my update of file paths. Updated now.